TUM Campus App/News.swift

```diff
@@ -11,14 +11,21 @@ import Sweeft
 
 final class News: DataElement {
     
+    enum Source: String {
+        case movie = "2"
+        case regular = "1"
```

Comment:
Source is a primary key in the table of the newsSources. Its currently right, that the ID 2 is the TU Film, but the ID can be anything. You should account for that as we add new sources in the future.

Here are all sources: 
![screenshot](https://user-images.githubusercontent.com/298860/32754603-a5d567e4-c8d1-11e7-96db-c359d92bdf8b.png)

Reply:
Ok. So do we expect the source ids to change over time?